src/modules/party/party.controller.test.ts

```diff
@@ -52,4 +62,35 @@ describe('PartyController', () => {
       expect(party).toStrictEqual(expectedParty);
     });
   });
+
+  describe('createParty', () => {
+    const { createPartyRequest: newParty } = new CreatePartyGenerator(valueGenerator, dateStringTransformations).generate({ numberToGenerate: 1 });
+    const res: Response = {
+      status: jest.fn().mockReturnThis(),
+    } as any;
+
+    it('creates a party with the service if the PartyAlternateIdentifier does not match existing parties', async () => {
```

Comment:
Do we need to add assertions where `newParty` has malformed payload?

Reply:
We have tests for malformed/invalid request in the API tests `post-parties.api-test.ts` as this would valid the validation of the DTO.